src/auto-complete.component.ts

```diff
@@ -124,6 +124,7 @@ export class NguiAutoCompleteComponent implements OnInit {
   @Input("show-dropdown-on-init") showDropdownOnInit: boolean = false;
   @Input("tab-to-select") tabToSelect: boolean = true;
   @Input("match-formatted") matchFormatted: boolean = false;
+  @Input("source-changes") sourceChanges: EventEmitter<any>;
```

Comment:
For any reason the source variable is not updating its value, so I've forced to do so using an event emitter who listen to external changes, that's why is an input.